test/engine/enginerpc/client.cc

```diff
@@ -134,5 +246,54 @@ int main(int argc, char** argv) {
     status = client.NewThresholdsFile(tf) ? 0 : 5;
     std::cout << "NewThresholdsFile: " << status << std::endl;
   }
+  else if (strcmp(argv[1], "GetNbrHost") == 0) {
```

Comment:
It is better to use strncmp, more secure
strncmp(argv[1], "GetNbrHost", strlen("GetNbrHost"));